### ISSUE TITLE
SOTG system improvements

### ISSUE THREAD

**Issue by dave0** [dave0/leaguerunner#21]
Originally filed by dave0000 on 2007-01-20T21:38:48
- ask the SOTG questions then present the SOTG number which the captain
  could change before submitting the score.
- each question could have a wider range of answers
- drop the highest and lowest SOTG from the average SOTG calculation
- investigate different ways to present SOTG (numeric or Xs & checks)

**Comment by dave0** [comment]
Updated by dave0000 on 2007-02-02T02:36:31

Change ownership to Tony

Original ticket set owner to tonysa13

**Comment by dave0** [comment]
Updated by tonysa13 on 2007-05-14T14:56:46

This has been fixed.  The spirit questions were changed to four: one about 
Timeliness, one about Rules Knowledge, one about Sportsmanship, and one about 
overall game Enjoyment.  The spirit reporting page now has an SOTG field where 
captains can enter any SOTG value they wish, or they can click a "Suggest" button, 
which will suggest a score based on the spirit question answers.  The high/low SOTG 
scores are dropped from the average.  And, SOTG information is available to all 
players for their team under their team's spirit link (this can be turned off by a 
league level configuration)...

Original ticket set status to Fixed (we converted to closed)
